test/index.html

```diff
@@ -64,7 +64,7 @@
   </script>
 
   <!-- The script under test -->
-  <script src="../inert.js"></script>
+  <script src="../dist/inert.js"></script>
```

Comment:
Yes. I figured if that's what we're actually distributing then it's what should be tested. The npm test commandwhich Travis uses  will now generate this build before running tests